feathr_project/feathr/api/app/routes/recommendation.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
```

Comment:
rename the file name to fit its content

Reply:
deleted, will have new route to handle search scenarios